app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+class CommentsController < ApplicationController
+  before_action :find_post, only: %i[new create save_like]
+
+  def new
+    @comment = Comment.new
+  end
+
+  def create
+    @comment = @post.comments.create(comment_params)
+    @comment.user_id = current_user.id
+    @comment.save
+
+    if @comment.save
+      redirect_to post_path(@post)
+    else
+      render 'new'
+    end
+  end
+
+  def save_like
```

Comment:
- [ ] It's better to [separate concerns](https://en.wikipedia.org/wiki/Separation_of_concerns) for creating and deleting likes. We can move this functionality into its own controller `likes_controller`. Creating or deleting a like doesn't really belong here.